Code/Framework/AzFramework/AzFramework/Spawnable/Script/SpawnableScriptMediator.h

```diff
@@ -79,5 +79,7 @@ namespace AzFramework::Scripts
         AZStd::recursive_mutex m_mutex;
         // used to track when SpawnableScriptMediator is destroyed to avoid executing logic in callbacks
         AZStd::shared_ptr<CallbackSentinel> m_sentinel;
+
+        AZStd::unordered_set<EntitySpawnTicket> m_activeSpawnTickets;
```

Comment:
nit: m_cachedSpawnTickets or m_createdSpawnTickets might be a better name since we can't be sure whether a created spawn ticket will be used by the user or not.